unity/Assets/Maroon/reusablePrefabs/NewVoltmeter/Scripts/VoltmeterLogic.cs

```diff
@@ -24,10 +24,16 @@ public float GetDifference()
 
             var currentDifference = startPinPotential - endPinPotential;
 
-            if (Mathf.Abs(currentDifference - currentValue.Value) > 0.0001)
+            if (Mathf.Abs(currentDifference - currentValue.Value) > 0.00000001)
                 currentValue.Value = currentDifference;
 
-            return currentDifference;
+            var check = currentValue.Value;
+            for (var cnt = 0; Mathf.Abs(check) < 1f && cnt < 2; ++cnt)
+            {
+                check *= Mathf.Pow(10, 3);
+            }
```

Comment:
This is a bit confusing to read IMO, maybe we can replace Mathf.Pow(10,3) simply with 1000 and make the loop a bit easier to understand

```
var returnValue = currentValue.Value;
for (int i = 0; i < 2 && Mathf.Abs(returnValue ) < 1f; i++)
{
    returnValue *= 1000;
}
return returnValue;
```